feedback/static/feedback.js

```diff
@@ -77,6 +78,108 @@ $(function() {
     });
   }
 
+  /* Buttons for toggling preview state */
+  function on_preview_button(e) {
+    var me = $(this);
+    var toolbar = me.closest('.btn-toolbar');
+    $('#' + toolbar.data('textarea-id')).each(function() {
+      var ta = $(this);
+      ta.trigger('preview_on');
+    });
```

Comment:
Good point. Personally I would have preferred to use `const`, but since the existing code used `var` in many instances where `const` should be used, I thought better to mimic that, but indeed it's probably better to use the proper way rather than mimic bad behavior.